qiskit/transpiler/passes/__init__.py

```diff
@@ -10,6 +10,12 @@
 from .unroller import Unroller
 from .cx_cancellation import CXCancellation
 from .fixed_point import FixedPoint
+from .resource_estimation import ResourceEstimation
+from .depth import Depth
+from .size import Size
+from .width import Width
+from .count_ops import CountOps
+from .num_tensor_factor import NumTensorFactor
```

Comment:
can you tweak the names here
```suggestion
from .num_tensor_factors import NumTensorFactors
```